.github/workflows/ci.yml

```diff
@@ -1,5 +1,5 @@
 # Do not edit these workflows directly as the changes made will be overwritten.
-# Instead, edit the template '.github/workflows/templates/ci.yml.jinja'
+# Instead, edit the template '.github\workflows\templates\ci.yml.jinja'
```

Comment:
Why are we changing from forward to back slashes?

Reply:
Sorry, when I regenerate the workflows, since I'm doing it on Windows, it changes those slashes